### ISSUE TITLE
KAPPA not defined

### ISSUE THREAD

**Issue by rndsrc** [AFD-Illinois/igrmonty#14]
On the latest master (commit 622ec174), compiling returns the error:

```
Undefined symbols for architecture x86_64:
  "_KAPPA", referenced from:
      _hypergeom_eval in kappa_sampler.o
      _dF3 in kappa_sampler.o
      _dF4 in kappa_sampler.o
      _dF5 in kappa_sampler.o
      _dF6 in kappa_sampler.o
      _F3 in kappa_sampler.o
      _F4 in kappa_sampler.o
      ...
```

Although `KAPPA` is declared in `src/decs.h`, it's not actually defined in any `*.c` file.

Adding `double KAPPA = 5.0;` near the beginning of `src/kappa_sampler.c` would allow the code to compile, although this may cause KAPPA from a parameter file be ignored when kappa model is enabled.

P.S. Note that, in the `feature/var-kappa` branch (commit 4613d19b), KAPPA is defined as macro in various models.  Maybe "desc.h" is not included in the model files so it didn't cause an error?

**Comment by gnwong** [comment]
I'm unable to reproduce this. Can you send me a zip file of your igrmonty directory? My guess is that you're using a different compiler that's perhaps stricter than whatever I'm using. The only time kappa_sampler.c functions are "used" is in compton.c:sample_beta_distr_y(...), which never actually gets called. 

Does the error persist if you remove that function from compton.c (delete it or comment it out) and then remove the kappa_sampler.c file? kappa_sampler.c was only used in debugging.

If the above doesn't work, could you email me a zip file of your full igrmonty/ directory?

**Comment by gnwong** [comment]
Addressed by 1f10aa5. Seems to have been a linker issue as above. Maybe move toward validating releases with clang too?
